jgrapht-ext/pom.xml

```diff
@@ -35,6 +35,19 @@
 					</execution>
 				</executions>
 				<configuration>
+					<transformers>
```

Comment:
Done (commit _d52394d509b9043698c191d052bed1eaa91b90f8_). Will create a separate PR for this issue.